src/types/onyx/Policy.ts

```diff
@@ -292,9 +292,6 @@ type Policy = OnyxCommon.OnyxValueWithOfflineFeedback<
         /** The approval mode set up on this policy */
         approvalMode?: ValueOf<typeof CONST.POLICY.APPROVAL_MODE>;
 
-        /** Whether the auto approval is enabled */
-        isAutoApprovalEnabled?: boolean;
```

Comment:
This property is right now, never returned from the backend to the client so we can remove it from the type